packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -232,7 +233,7 @@ export async function createTemporaryTsConfig(includedPackageDirs: string[]) {
     extends: './tsconfig.json',
     include: [
       // These two contain global definitions that are needed for stable API report generation
-      'packages/cli/asset-types/asset-types.d.ts',
+      resolvePackagePathBackend('@backstage/cli', 'asset-types/asset-types.d.ts'),
```

Comment:
Oh. Hm. Shouldn't this file exist within the repo-tools package then? @Rugvip opinions?

Reply:
So, this function is creating a tmp tsconfig in the root and then it wants to include the `asset-types/asset-types.d.ts` file. But if you have your own Backstage repository you won't have access to the `backstage-repos` tool. So, eventually the path will resolve to: `/Users/djamaile/Desktop/projects/backstage/node_modules/@backstage/cli/asset-types/asset-types.d.ts` . Then every Backstage instance will be able to access it 😄

Reply:
And the repo-tools package doesn't depend on the cli either, not even as a peer dep. This is a bit confusing to me :)

Reply:
ohh good point, I didn’t noted that... maybe move it to `cli-common` then? or duplicate the file?

Reply:
I think duplicating the file will break the CI. If we can add `@backstage/cli` as a dependency, that would be best. Question is what type of dependency makes sense here?

Reply:
We could make it an optional peer dep, but tbh I'm feeling we could just optimistically try to resolve it an fall back to ignoring. I'm kinda fine with shipping it as is, but for added safety we could wrap the resolve up in a try catch.